lib/ui/home.dart

```diff
@@ -212,8 +213,20 @@ class Home extends StatelessWidget {
     );
   }
 
-  //TODO Build it properly
-  TabNavigator _buildTabNavigatorWithCircleImageWidget(Widget page) {
+  Widget _buildProfileIcon(HomeViewModel viewModel) {
+    return ProfileAvatar(
+      viewModelProvider: ProfileDetailProvider(
+          profileRepo: viewModel.currentProfile,
+          plotRepo:
+              repositoryProvider.getMyPlotRepository(viewModel.currentProfile)),
+      width: _Constants.iconHeight,
```

Comment:
Is this correct or should be iconWidth?

Reply:
Probably this builds a circle avatar with same width and height for the image, so the width and height would be the same size. Maybe I would rename it to `iconSize`?

Reply:
just used existing code. I´ll rename tho

Reply:
done 👍